creator-node/src/middlewares.js

```diff
@@ -52,7 +52,9 @@ async function syncLockMiddleware (req, res, next) {
 
 /** Blocks writes if node is not the primary for audiusUser associated with wallet. */
 async function ensurePrimaryMiddleware (req, res, next) {
-  if (config.get('isUserMetadataNode')) next()
+  if (config.get('isUserMetadataNode')) {
+    return next()
```

Comment:
without calling `return` here code continues executing this function after calling next(), surprised we didn't come across this earlier
https://stackoverflow.com/questions/16810449/when-to-use-next-and-return-next-in-node-js